modules/mono/glue/GodotSharp/GodotSharp.UnitTest/Vector2iTests.cs

```diff
@@ -0,0 +1,59 @@
+using Godot;
+using NUnit.Framework;
+
+namespace GodotSharp.UnitTest
+{
+    public class Vector2iTests
```

Comment:
Vector2i does not even exist. However, these tests aren't being ran so this is currently 'floating' around.

Reply:
It's right here: https://github.com/godotengine/godot/blob/master/modules/mono/glue/GodotSharp/GodotSharp/Core/Vector2i.cs

Reply:
Whoops got the c# side and the C++ side mixed up